lms/static/scripts/frontend_apps/components/InstructorToolbar.tsx

```diff
@@ -26,20 +38,26 @@ export default function InstructorToolbar() {
           className="text-lg font-semibold leading-none"
           data-testid="assignment-name"
         >
-          {grading.assignmentName}
+          {assignmentName}
         </h1>
         <h2
           className="text-sm font-normal text-color-text-light leading-none"
           data-testid="course-name"
         >
-          {grading.courseName}
+          {courseName}
         </h2>
       </div>
 
+      {editingEnabled && (
+        <LabeledButton onClick={() => alert('Not implemented yet')}>
+          Edit
+        </LabeledButton>
+      )}
+
```

Comment:
Hey @robertknight You don't need to get the correct component here yet, but can you please use a non-deprecated one from `@hypothesis/frontend-shared/lib/next`? This component has already had a pass for removing deprecated components. You could use `Button`; it's a drop-in replacement for `LabeledButton`.